Strange, didn't seem to take:
```
GCC version
gcc (Ubuntu/Linaro 4.6.3-1ubuntu5) 4.6.3
```

```
../node_modules/nan/nan.h:43:3: error: #error This version of node/NAN/v8 requires a C++11 compiler
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,9 +1,15 @@
-before_install:
-  - sudo apt-get update -qq
-  - sudo apt-get install -y libavahi-compat-libdnssd-dev
 language: node_js
 node_js:
-  - "0.12"
-  - "0.11"
-  - "0.10"
-  - "iojs"
+  - "4"
+  - "6"
+  - "7"
+# https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Node.js-v4-(or-io.js-v3)-compiler-requirements
+env:
+  - CXX=g++-4.8
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+      - libavahi-compat-libdnssd-dev

